xio-core/src/test/java/com/xjeffrose/xio/http/ClientTest.java

```diff
@@ -0,0 +1,112 @@
+package com.xjeffrose.xio.http;
+
+import static io.netty.handler.codec.http.HttpMethod.GET;
+import static org.mockito.Mockito.times;
+import static org.powermock.api.mockito.PowerMockito.when;
+
+import com.typesafe.config.ConfigFactory;
+import com.xjeffrose.xio.bootstrap.ChannelConfiguration;
+import com.xjeffrose.xio.client.ClientConfig;
+import com.xjeffrose.xio.tracing.HttpClientTracingHandler;
+import com.xjeffrose.xio.tracing.XioTracing;
+import io.netty.buffer.Unpooled;
+import io.netty.channel.*;
+import lombok.val;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.*;
+
+public class ClientTest extends Assert {
+
+  private Client subject;
+  @Mock private XioTracing tracing;
+  @Mock private HttpClientTracingHandler tracingHandler;
+
+  ChannelHandler appHandler =
+      new ChannelHandler() {
+        @Override
+        public void handlerAdded(ChannelHandlerContext channelHandlerContext) throws Exception {}
+
+        @Override
+        public void handlerRemoved(ChannelHandlerContext channelHandlerContext) throws Exception {}
+
+        @Override
+        public void exceptionCaught(
+            ChannelHandlerContext channelHandlerContext, Throwable throwable) throws Exception {}
+      };
+
+  @Before
+  public void setUp() {
+    MockitoAnnotations.initMocks(this);
+  }
+
+  @After
+  public void tearDown() {
+    Mockito.reset(tracing);
+    Mockito.reset(tracingHandler);
+  }
+
+  @Test
+  public void testDisabledTracing() {
+    val channelConfig = ChannelConfiguration.clientConfig(1, "worker");
+    val clientConfig =
+        new ClientConfig(ConfigFactory.load().getConfig("xio.invalidZipkinParameters"));
```

Comment:
I don't see any changes to `ClientConfig` to handle zipkin settings

Reply:
The zipping settings are in the constructor of XioTracing.
XioTracing takes a Config and configures itself.
Unless we still want to do the .conf check inside ClientConfig and directly construct XioTracing(ZipkinUrl, SamplingRate)

Reply:
I will use a standard config instead since we don't actually need the xio.invalidZipkin settings to create a ClientConfig

Reply:
I'm confused, ClientConfig isn't parsing the zipkin values in `xio.invalidZipkinParameters`, so what's under test here?

Reply:
as per our chat earlier, ClientConfig won't have any zipkin settings

Reply:
fixed